packages/aws-lambda-otel-extension/opt/otel-extension/internal/index.js

```diff
@@ -329,6 +329,28 @@ const instrumentations = [
         eventData[context.awsRequestId].eventCustomRequestTimeEpoch =
           event.requestContext.timeEpoch;
       }
+
+      // Send request data to external so that we can attach this data to logs
+      await fetch(`http://localhost:${process.env.MOCK_PORT || OTEL_SERVER_PORT}`, {
```

Comment:
This is just copying behavior from the other fetch happing in this file. If I am remembering correctly it is because we set the port to something different during one of the tests 🤷

Reply:
We don't set MOCK_PORT in any test. It looks as some never used left over

Reply:
`MOCK_PORT` is set in the `packages/aws-lambda-otel-extension/test/unit/internal/index.test.js` file

Reply:
Indeed, sorry I didn't realize that (by mistake I've search in context of extension, ignoring test files)

Reply:
All good and looking into this more I don't see why we couldn't use the already defined `OTEL_SERVER_PORT` so I went ahead and removed this `MOCK_PORT` nonsense and the unit tests still pass 👍

Reply:
Great call, thanks @Danwakeem ! 👍